src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseController.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.iverksett.arena
+
+
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(path = ["/api/iverksett"])
+@ProtectedWithClaims(issuer = "azuread")
+class VedtakhendelseController(
```

Comment:
Er denne endepunkt for testing eller hva er flytet for å sende vedtakshendelse til arena?  (trenger vi en endepunkt eller er det bare en task som skal utføres ved iverksetting)

Reply:
Sending av vedtaksstatistikk kan kjøres i iverksett-flyten, så det er bare et endepunkt for testing. Lager egen task med hva som skal sendes på MQ-kø i egen PR.